include/zephyr/bluetooth/gatt.h

```diff
@@ -167,16 +167,57 @@ typedef ssize_t (*bt_gatt_attr_write_func_t)(struct bt_conn *conn,
 
 /** @brief GATT Attribute structure. */
 struct bt_gatt_attr {
-	/** Attribute UUID */
+	/** @brief Attribute Type, aka. "UUID" */
 	const struct bt_uuid *uuid;
-	/** Attribute read callback */
+
+	/** @brief Attribute Value function
+	 *
+	 * This is the general API to read from the Attribute Value of
```

Comment:
I believe it is `general -> generic`. Similar in other places
```suggestion
	 * This is the generic API to read from the Attribute Value of
```